### PR TITLE
MINOR: Dynamodb sample data

### PR DESCRIPTION
**Pull request by sushi30**
### Describe your changes:

Add support for DynamoDB sample data.

Builds on:

- https://github.com/open-metadata/OpenMetadata/pull/15204
- https://github.com/open-metadata/OpenMetadata/pull/15237

image

#
### Type of change:

- [x] Improvement

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

- [ ] I have added tests around the new logic.
- [ ] For connector/ingestion changes: I updated the documentation.

### CODE REVIEW

File: ingestion/src/metadata/profiler/adaptors/factory.py
Comment (IceS2) [inline ingestion/src/metadata/profiler/adaptors/factory.py:75]: Pretty small nitpick here:
I believe it'd be best if we keep the same way of actually getting the name of the adaptor. Maybe we could have a 
```
@property
@classmethod
def name():
```

or something like that. What do you think?

**Review by IceS2 — APPROVED** [review]

**Comment by sushi30** [comment]
@IceS2 please re-approve after merging and fixing tests

**Review by IceS2 — APPROVED** [review]

**Pull request merged**
